x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_form.tsx

```diff
@@ -471,6 +491,22 @@ export const ActionTypeForm = ({
                           </EuiBadge>
                         </EuiFlexItem>
                       )}
+                      {warning && !isOpen && (
+                        <EuiFlexItem grow={false}>
+                          <EuiBadge data-test-subj="warning-badge" iconType="alert" color="warning">
+                            {i18n.translate(
+                              'xpack.triggersActionsUI.sections.actionTypeForm.actionWarningsTitle',
+                              {
+                                defaultMessage:
+                                  '{warningsToShow} {warningsToShow, plural, one {warning} other {warnings}}',
+                                values: {
+                                  warningsToShow: 1,
```

Comment:
nit: we only ever have 1 warning so maybe we don't need this plural logic here right now?

Reply:
Resolved in this commit https://github.com/elastic/kibana/pull/149832/commits/a3033820c4a161745d9e77ebb9d81e6ea254fa2c